recipe/meta.yaml

```diff
@@ -22,33 +22,40 @@ requirements:
 
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
```

Comment:
`setuptools` is no more required as a build backend, see  https://github.com/mwaskom/seaborn/blob/231f1ba61cffdd2de8d27c3ad06b1bd841ecac63/pyproject.toml#L3 so it  can be removed